docs/csharp/tutorials/working-with-linq.md

```diff
@@ -35,26 +35,23 @@ If you've never used C# before, [this tutorial](console-teleprompter.md) explain
 
 ## Creating the Data Set
 
-Let's start by creating a deck of cards. You'll do this using a LINQ query that has two sources (one for the four suits, one for the thirteen values). You'll combine those sources into a 52 card deck. A `Console.WriteLine` statement inside a `foreach` loop displays the cards.
-
-Here's the query:
+Before we begin, make sure that the following lines are at the top of the `Program.cs` file generated by `dotnet new console`:
 
 ```csharp
-var startingDeck = from s in Suits()
-                   from r in Ranks()
-                   select new { Suit = s, Rank = r };
-
-foreach (var c in startingDeck)
-{
-    Console.WriteLine(c);
-}
+// Program.cs
+using System;
+using System.Collections.Generic;
+using System.Linq;
 ```
 
-The multiple `from` clauses produce a `SelectMany`, which creates a single sequence from combining each element in the first sequence with each element in the second sequence. The order is important for our purposes. The first element in the first source sequence (Suits) is combined with every element in the second sequence (Values). This produces all thirteen cards of first suit. That process is repeated with each element in the first sequence (Suits). The end result is a deck of cards ordered by suits, followed by values.
+If these three lines (`using` statements) aren't at the top of the file, our program will not compile.
 
-Next, you'll need to build the Suits() and Ranks() methods. Let's start with a really simple set of *iterator methods* that generate the sequence as an enumerable of strings:
+Now that we have all of the references we need, let's consider what constitutes a deck of cards. Commonly, a deck of playing cards has four suits, and each suit has thirteen values. Normally, you might consider creating a `Card` class right off the bat and populating a collection of `Card` objects by hand. With LINQ, we can be more concise than the usual way of dealing with creating a deck of cards. Instead of creating a `Card` class, let's use two collections that will represent suites and ranks, respectively. We'll create a really simple pair of [*iterator methods*](../iterators.md#enumeration-sources-with-iterator-methods) that will generate the ranks and suits as enumerables of strings:
```

Comment:
Will change as well. My thought process for saying "let's consider" over "consider was that it would be more conversational rather than a textbook-like "consider" (i.e., a calculus textbook that would say "Consider the formula {blah}").